src/utils/theme.js

```diff
@@ -30,9 +30,14 @@ export const breakpoints = {
   desktop: 1197
 }
 
+export const elementSizes = {
```

Comment:
why is this exported?

Reply:
good question, I thought it was our common practice. But we can live without it :)